docs/source/modin/index.rst

```diff
@@ -0,0 +1,21 @@
+===================
+Snowpark pandas API
+===================
+
+This page gives an overview of all public Snowpark pandas objects, functions and methods.
+For your convenience, here is all the :doc:`Supported APIs <supported/index>`
+
+
+.. toctree::
+    :maxdepth: 2
+
+    session
+    io
+    general_functions
+    modin.pandas.Series <series>
```

Comment:
The plan is to try to have all of the modin documentation displayed as `modin.pandas.<class>` instead of `snowflake.snowpark.modin.pandas.<class>` - this is a WIP and I'll introduce that in a new PR.